ui/src/app/data-explorer/models/dataview-dashboard.model.ts

```diff
@@ -104,6 +104,12 @@ export interface FieldProvider {
     nonNumericFields: DataExplorerField[];
 }
 
+export interface AxisConfig {
+    axisScaleMode: boolean;
```

Comment:
What is the axisScaleMode? what do you think about renaming it to `autoScaleActive`?